site-manage/site-manage-tool/tool/src/java/org/sakaiproject/site/tool/SiteAction.java

```diff
@@ -15325,16 +15325,11 @@ private String prefixString(String prefix, String aliasId) {
 	 */
 	private boolean isHomePage(SitePage page)
 	{
-		if (page.getProperties().getProperty(SitePage.IS_HOME_PAGE) != null)
-		{
-			// check based on the page property first
-			return true;
-		}
-		else
-		{
-			// if above fails, check based on the page title
-			String pageTitle = page.getTitle();
-			return TOOL_ID_HOME.equalsIgnoreCase(pageTitle) || rb.getString("java.home").equalsIgnoreCase(pageTitle);
+		//removed "check by title" : that creates unexpected results with normal pages titled as "HOME"
+		try {
+			return page.getProperties().getBooleanProperty(SitePage.IS_HOME_PAGE);
```

Comment:
I think you can just do:

    return page.isHomePage()

and drop the try/catch.